Makefile

```diff
@@ -472,23 +471,24 @@ proto:
 	$(PROTOC) \
 		$(PROTO_INCLUDES) \
 		--gogo_out=plugins=grpc,$(PROTO_GOGO_MAPPINGS):$(PWD)/proto-gen/ \
+		--swagger_out=allow_merge=true:$(PWD)/proto-gen/openapi/ \
```

Comment:
let's not do the swagger

Reply:
Just curious, why not?

Reply:
because it gives the impression that it is somehow the officially supported JSON, which is not the case.